main.js

```diff
@@ -136,41 +149,54 @@ function changeTimerColor() {
 
 function startTimer() {
   startTimerButton.disabled = true;
-  setInterval(function () {
+  var interval = setInterval(function () {
     if (currentActivity.completed === false) {
-      currentActivity.countdown()   
-      //   if (currentActivity.completed === true) {
-    //     showElement(logActivityButton)
-    //   }
-    // }
+      currentActivity.countdown()
+    } else {
+      clearInterval(interval);
     }
   }, 1000)
 };
 
 
 
-function showMyLogs() {
-  activityCards.forEach(card => {
-    pastActivityCards.innerHTML = `
-    <div class="card-container">
-      <div class="card-header">
-        <p class="card-category">${currentActivity.category}</p>
-        <p class="card-duration">${currentActivity.minutes} MIN   :   ${currentActivity.seconds} SEC</p>
-        <p class="card-body">${currentActivity.description}</p>
+function saveActivityToLocalStorage() {
+  localStorage.setItem(`${currentActivity.id}`, JSON.stringify(currentActivity));
+  pastActivityCards.innerHTML = '';
+  showMyLog();
+}
+
+
+function showMyLog() {
+  for (i = 0; i < localStorage.length; i++) {
+    pastActivityCards.innerHTML += `
+    <div class='card-container'>
+      <div class='card-header'>
+        <p class='card-category'>${JSON.parse(localStorage.getItem(localStorage.key(i))).category}</p>
+        <p class="card-duration">${JSON.parse(localStorage.getItem(localStorage.key(i))).savedMinutes} MIN : ${adjustSeconds(JSON.parse(localStorage.getItem(localStorage.key(i))).savedSeconds)} SEC</p>
+        <p class="card-body">${JSON.parse(localStorage.getItem(localStorage.key(i))).description}</p>
+      </div>
+        <div class = "sliver-${JSON.parse(localStorage.getItem(localStorage.key(i))).category}">
       </div>
-      <div class="sliver"></div>
     </div>
     `
-    if (currentActivity.category === 'study') {
-      document.querySelector('.sliver').classList.add('sliver-green')
-    } else if (currentActivity.category === 'meditate') {
-      document.querySelector('.sliver').classList.add('sliver-purple')
-    } else if (currentActivity.category === 'exercise') {
-      document.querySelector('.sliver').classList.add('sliver-red')
+    if (JSON.parse(localStorage.getItem(localStorage.key(i))).category === 'study') {
+      document.querySelector(".sliver-study").classList.add('sliver-green');
+    } else if (JSON.parse(localStorage.getItem(localStorage.key(i))).category === 'meditate') {
+      document.querySelector(".sliver-meditate").classList.add('sliver-purple');
+    } else if (JSON.parse(localStorage.getItem(localStorage.key(i))).category === 'exercise') {
+      document.querySelector('.sliver-exercise').classList.add('sliver-red');
```

Comment:
Since we're using a for loop above, we won't need this. Line 179 above already does it. We just need to adjust the css to say:
`.sliver-meditate {
  background: #C278FD;
  border-right: 2px solid #C278FD;
  border-left: 2px solid #C278FD;
}

.sliver-study {
  background: #B3FD78;
  border-right: 2px solid #B3FD78;
  border-left: 2px solid #B3FD78;
}

.sliver-exercise {
  background: #FD8078;
  border-right: 2px solid #FD8078;
  border-left: 2px solid #FD8078;
}`